test.js

```diff
@@ -14,6 +14,10 @@ function commit (msg) {
   shell.exec('git commit --allow-empty -m"' + msg + '"')
 }
 
+function execCli (argString) {
+  return shell.exec('node ' + cliPath + ' ' + argString)
```

Comment:
Should change this to:

```js
return shell.exec('node ' + cliPath + (argString ? ' ' + argString : ''))
```

To avoid command strings like `node /Users/nexdrew/git/qi/standard-version/index.js undefined`